subsys/bluetooth/controller/ll_sw/ull_adv_aux.c

```diff
@@ -595,10 +601,49 @@ uint8_t ll_adv_aux_set_remove(uint8_t handle)
 
 uint8_t ll_adv_aux_set_clear(void)
 {
-	/* TODO: reset/release all adv set primary channel and  Aux channel
-	 * PDUs
-	 */
-	return 0;
+	struct ll_adv_set *adv;
```

Comment:
Any reasons why `ll_adv_aux_set_remove`  cant be called in a loop?

Reply:
Changed to call ll_adv_aux_set_remove in a loop.